### ISSUE TITLE
services: Incorrect color usage for error state rows (needs better contrast)

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#13944]
It's hard to read the text in the new services page on units with errors:

![Screenshot_2020-04-22 Services - debian](https://user-images.githubusercontent.com/10246/79983088-8f7d9b80-84a7-11ea-95a7-c1fa5ab6b481.png)

The colors of the `.service-unit-failed` class needs updating, as does `.service-unit-status-failed` (most likely).

Alternatively (or additionally), `--color-ct-list-critical-bg` is currently used and might need lightening.

**Comment by KKoukiou** [comment]
This exact same CSS is used also in machines page for errors; so if we will fix it we need to fix it in both places.

**Comment by garrett** [comment]
@KKoukiou: OK! Thanks! Good to know. (I should have caught this way earlier.)

We'll aim for AA contrast.

**Comment by garrett** [comment]
First-pass improved background combos:

- Based on #06C, the background should be #FBE2E2 (instead of #F8CCCC) for AA and then _also_ change blue #004A95 to get to AAA.
- Based on the new background, the foreground red for the warning text should be #D30000 (instead of #F00) for AA or #9D0000 for AAA.

However, these need to be adjusted for the PF4 color palette space.

---

After attempting to use the Sketch file (importing into a Figma demo account), I realized they didn't have a palette! So I scraped the website, exported the data to a text file with hex colors, and imported it into LibreOffice Calc (spreadsheet). Then I wrote some simple functions to convert the hex to red, green, and blue components, as that's what GIMP's palette format supports. After getting the results I wanted, I exported _back_ to text and polished it up so that GIMP would recognize it as a palette.

We now have the PatternFly 4 extended palette for both GIMP and Inkscape; I've added it to our design repo @ https://github.com/cockpit-project/cockpit-design/blob/master/PatternFly4_Palette-extended.gpl (it's 80 colors!)

Having the palette is useful for myself, Andreas, Mo, and others. I've put off the effort before, but finally ran through to properly fix this issue. :partying_face: 

---

OK, _after all that,_ I used GIMP to quantize the colors I found above to map it back to PF colors and got:

AA:

- background: #faeae8 → red-50
- alert text red: #c9190b → red-100

AAA: 

- background: #faeae8 → red-50
- link (dark) #004080 → blue-500
- alert text (dark): #a30000 → red-200

---

I'll use this info and send a PR up to fix this soon.
